reflect.js

```diff
@@ -1443,6 +1444,24 @@ Object.defineProperty = function(subject, name, desc) {
   }
 };
 
+Object.defineProperties = function(subject, descs) {
+  var vhandler = directProxies.get(subject);
+  if (vhandler !== undefined) {
+    var names = prim_getOwnPropertyNames(descs);
```

Comment:
The ES5 spec mentions that only the enumerable own properties of the `descs` object are considered. So you should use `Object.keys` here. Also, `descs` might be a proxy so it's best to call the patched `Object.keys`, as `prim_keys` will not work when `descs` is a proxy.